tools/cmgen/src/cmgen.cpp

```diff
@@ -1164,7 +1172,7 @@ void iblLutDfg(utils::JobSystem& js, const utils::Path& filename, size_t size, b
 
 void extractCubemapFaces(utils::JobSystem& js, const utils::Path& iname, const Cubemap& cm,
         const utils::Path& dir) {
-    utils::Path outputDir(dir.getAbsolutePath() + iname.getNameWithoutExtension());
```

Comment:
All cubemap faces won't be extracted into directory anymore?

Reply:
Goot catch, this needs to be changed to affect KTX only.

Reply:
Fixed.